packages/dashboard/src/components/FooterLine/Version.tsx

```diff
@@ -0,0 +1,19 @@
+import { useDashboardI18N } from '../../locales'
+import { Typography } from '@material-ui/core'
+
+export const Version = ({ className }: { className?: string }) => {
+    const t = useDashboardI18N()
+    const version = globalThis.browser?.runtime.getManifest()?.version ?? process.env.TAG_NAME.slice(1)
+
+    return (
+        <Typography className={className} variant="body2" component="span" color="inherit" fontSize={'inherit'}>
```

Comment:
Why is it necessary to set `fontSize` when ` variant="body2"` is already used here? Did the designer not provide the text style?

Reply:
It's not necessary. I will remove it.